src/main/java/seedu/address/model/loan/LoanStatus.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.model.loan;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Represents a LoanStatus in the LoanBook.
+ */
+public enum LoanStatus {
+    ONGOING {
+        public String toString() {
+            return "Ongoing";
+        }
+    },
+    RETURNED {
+        public String toString() {
+            return "Returned";
+        }
+    },
+    DELETED {
+        public String toString() {
+            return "Deleted";
+        }
+    };
```

Comment:
Question: Why do we need a "Deleted" Loan Status?

Is it because when we're "deleting" a Loan, we're not actually deleting it, but simply marking it as "DELETED"?

Reply:
Yes, that is true. This is in preparation for the undo delete feature in v2.0.